apps/application/flow/step_node/document_extract_node/impl/base_document_extract_node.py

```diff
@@ -70,7 +70,7 @@ def save_image(image_list):
                     # 回到文件头
                     buffer.seek(0)
                     file_content = split_handle.get_content(buffer, save_image)
-                    content.append('## ' + doc['name'] + '\n' + file_content)
+                    content.append('### ' + doc['name'] + '\n' + file_content)
                     break
 
         return NodeResult({'content': splitter.join(content)}, {})
```

Comment:
There is a minor error in the code that needs correction:

Replace this line:
`content.append('## ' + doc['name'] + '\n' + file_content)`

 With this line:
 `content.append(f"### {doc['name']}\n{file_content}")`

Specifically, you should use an f-string to escape the double quotes properly within your string interpolation. This will ensure proper formatting when adding each document's name and content.

Also, while there is no significant issue with the rest of the function (such as improper handling exceptions or inefficiencies), it doesn't require any major optimizations for its current implementation. However, make sure that the variable names (`buffer`, `split_handle`, `splitter`) are appropriate for their usage context and adhere to common coding conventions for clarity and maintainability.